src/http.rs

```diff
@@ -12,44 +14,54 @@ pub fn retrieve_asset(
     opt_insecure: bool,
 ) -> Result<(String, String), reqwest::Error> {
     if is_data_url(&url).unwrap() {
+        eprintln!("is data URL [ {} ]", &url);
```

Comment:
we may need to leave this out of this PR

Reply:
yes, sorry... I forgot to remove this